packages/sources/src/components/navbar/navbar.stories.ts

```diff
@@ -0,0 +1,85 @@
+import { bindTemplate, componentArgs, StorybookParameters } from '../../stories-helpers';
+
+import { NavbarArgs, navbarArgsMapper, navbarArgTypes } from './navbar.args';
+import { Navbar } from './navbar.models';
+
+export interface NavbarParameters<TemplateFnReturnType> {
+  navbarTemplate: (navbarProperties: Navbar) => TemplateFnReturnType;
+}
+
+export function storiesOfNavbar<TemplateFnReturnType>(
+  {
+    module: mainModule,
+    storiesOf,
+    readme
+  }: StorybookParameters,
+  {
+    navbarTemplate
+  }: NavbarParameters<TemplateFnReturnType>
+) {
+  const template = bindTemplate(navbarArgsMapper, navbarTemplate);
+
+  const stories = storiesOf('Navbar', mainModule)
+    .addParameters({
+      docs: {
+        page: readme
+      },
+      argTypes: navbarArgTypes,
+      args: componentArgs<NavbarArgs>({
+        href: '#',
+        open: false,
+        modifier: '',
+        items: []
+      })
+    });
+
+  stories.add(
+    'primary',
+    template,
+    {
+      args: componentArgs<Pick<NavbarArgs, 'modifier' | 'items'>>({
+        modifier: 'main',
+        items: [
+          {
+            label: 'Home',
+            active: true
+          },
+          {
+            label: 'Vergunningcheck'
```

Comment:
ik verwacht hier in ieder item een href (al is het maar een `#`)

Reply:
idem als hierboven